hudi-cli/src/main/java/org/apache/hudi/cli/HoodiePrintHelper.java

```diff
@@ -84,7 +84,7 @@ private static String print(Table buffer) {
     buffer.getFieldNames().toArray(header);
 
     String[][] rows =
-        buffer.getRenderRows().stream().map(l -> l.stream().toArray(String[]::new)).toArray(String[][]::new);
+        buffer.getRenderRows().stream().map(l -> l.toArray(new String[0])).toArray(String[][]::new);
```

Comment:
why `new String[0]`?

Reply:
This should be pre-size for `toArray` method, and I have already modified.